policyengine_us/parameters/gov/states/ca/cdss/tanf/cash/resources/sources.yaml

```diff
@@ -0,0 +1,16 @@
+description: California CalWORKs counts these property sources as resources.
+
+metadata:
+  unit: list
+  period: year
+  label: California CalWORKs property breakdown
+  reference:
+    - title: Los Angeles Department of Public Social Services E-Policy | Property Definitions
+      href: http://epolicy.dpss.lacounty.gov/epolicy/epolicy/server/general/projects_responsive/ePolicyMaster/index.htm?&area=general&type=responsivehelp&ctxid=&project=ePolicyMaster#t=mergedProjects%2FCalWORKs%2FCalWORKs%2F42-200_Property%2F42-200_Property.htm%23Definitionsbc-4&rhtocid=_3_1_2_0_3
+values:
+  2024-01-01:
+    - real_property
```

Comment:
I think this would be grouped as property value

Reply:
The reference distinguishes between each type of property (as these are the resource constituents). Should we do the same or combine into one?

Reply:
We should combine them